src/modules/LanItems.rb

```diff
@@ -485,6 +485,9 @@ def update_item_udev_rule!(based_on = :mac)
     # @param new_val     [string] value for new key
     # @return updated rule when replace_key is found, current rule otherwise
     def ReplaceItemUdev(replace_key, new_key, new_val)
+      Items()[@current]["udev"] = { "net" => [] } if !Items()[@current]["udev"]
```

Comment:
well I believe that this line is not needed bcs of this https://github.com/yast/yast-network/blob/master/src/modules/LanItems.rb#L1085